### PR TITLE
Fix status page

### PR DESCRIPTION
**Pull request by MetRonnie**
- Try using HTML table to see if that fixes render on GH Pages
- Fix cylc-flow listed workflows

### CODE REVIEW

File: docs/status/template.md.jinja2 [hunk 1]
```diff
@@ -1,14 +1,5 @@
-{#
-
-This template/markdown file is auto-generated, please do not edit it directly.
-
-Please edit `branches.json` and raise a pull request, this fille will be
-automatically regenerated on merge.
-
-#}
```
Comment (MetRonnie) [inline docs/status/template.md.jinja2:8]: I've moved this comment to the auto-generate file

File: docs/status/branches.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
         }
     },
     "branch_actions": {
-        "cylc/cylc-flow": ["test_fast", "test_functional", "build", "test_conda-build"],
+        "cylc/cylc-flow": ["test_fast", "test_functional", "bash", "test_manylinux", "test_conda-build", "test_tutorial_workflow"],
```
Comment (oliver-sanders) [inline docs/status/branches.json:19]: "build" removed purposefully?

File: docs/status/template.md.jinja2 (hunk 1 above)
Comment (oliver-sanders) [reply docs/status/template.md.jinja2:8]: Why? Not much point putting it there!

File: docs/status/template.md.jinja2 [hunk 3]
```diff
@@ -1,14 +1,5 @@
...
-
-{% macro badge(repo, branch, action) -%}
-[![{{ action }}](https://github.com/{{ repo }}/actions/workflows/{{ action }}.yml/badge.svg?branch={{ branch }})](https://github.com/{{ repo }}/actions/workflows/{{ action }}.yml)
+{% macro badge(repo, branch, workflow) -%}
```
Comment (oliver-sanders) [inline docs/status/template.md.jinja2:1]: Why rename every instance of action to workflow?!?!?!

Comment (MetRonnie) [reply docs/status/template.md.jinja2:1]: Because they're workflows not actions

File: docs/status/branches.json (hunk 2 above)
Comment (MetRonnie) [reply docs/status/branches.json:19]: There isn't a `build.yml` workflow

File: docs/status/template.md.jinja2 (hunk 1 above)
Comment (MetRonnie) [reply docs/status/template.md.jinja2:8]: But isn't that where it was meant to go? The template isn't the one that's autogenerated

File: docs/status/template.md.jinja2 (hunk 3 above)
Comment (oliver-sanders) [reply docs/status/template.md.jinja2:1]: Yes and no. With this change there's now a confusing mix of terminology within the same file which is much worse.

File: docs/status/branches.json (hunk 2 above)
Comment (oliver-sanders) [reply docs/status/branches.json]: MB, the workflow was added on a PR which didn't get merged so it showed in GH but wasn't on master. I've cherry-picked that one onto master so it's there again now.

File: docs/status/template.md.jinja2 (hunk 1 above)
Comment (oliver-sanders) [reply docs/status/template.md.jinja2:8]: Derp.

File: docs/status/branches.json (hunk 2 above)
Comment (MetRonnie) [reply docs/status/branches.json]: Yeah I saw, added back in after rebase

File: docs/status/template.md.jinja2 (hunk 3 above)
Comment (MetRonnie) [reply docs/status/template.md.jinja2:1]: Is it too late to rename `nightly_actions` and `branch_actions`?

Comment (oliver-sanders) [reply docs/status/template.md.jinja2:1]: If you really want to go through all the downstream workflows, but there are much more worthwhile uses of our time.

**Pull request merged**